WeakAuras/RegionTypes/StopMotion.lua

```diff
@@ -143,8 +154,9 @@ end
 
 local function modify(parent, region, data)
     WeakAuras.regionPrototype.modify(parent, region, data);
-
+    region.foreground = region.foreground or {}
     local pattern = "%.x(%d+)y(%d+)f(%d+)%.[tb][gl][ap]"
+    local pattern2 = "%.x(%d+)y(%d+)f(%d+)w(%d+)h(%d+)W(%d+)H(%d+)%.[tb][gl][ap]"
     local tdata = texture_data[data.foregroundTexture];
     if (tdata) then
```

Comment:
You need to ensure that all 3 differnt configuration options set all properties.

That is consider the case where someone first selects a pattern2 file, and then switches to one of the other configuration methods.